src/build_backend.py

```diff
@@ -70,8 +70,9 @@ def build_wheel(wheel_directory: str,
         ],
         'entry_points.txt': [
             '[console_scripts]',
-            'cockpit-bridge = cockpit.bridge:main',
             'cockpit-askpass = cockpit._vendor.ferny.interaction_client:main',
+            'cockpit-beiboot = cockpit.beiboot:main',
```

Comment:
I like how you handled this.

For the record, I've entertained an idea in the past of having a (possibly secret) mode in the bridge that supports invoking a Python module instead of an executable.  Then we could launch `cockpit.beiboot` directly and skip the wrapper.  I'd sort of like to avoid having stuff in libexecdir in the long run...